tools/python/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ set(python_srcs
    src/shape_predictor.cpp
    src/correlation_tracker.cpp
    src/face_recognition.cpp
+   src/face_detection.cpp
```

Comment:
This should be renamed to cnn_face_detector.cpp.  Otherwise, people looking for the HOG face detector code will probably be confused that it's not in this file.